docs/protocol/simplex-chat.md

```diff
@@ -213,6 +213,8 @@ Currently members can have one of three roles - `owner`, `admin` and `member`. T
 
 `x.grp.mem.info` this message is sent as part of member connection handshake - it includes group member profile.
 
+`x.grp.mem.role` message is sent to update group member role - it is sent to all members by the member who updated the role of the member referenced in this message. This message MUST only be sent by members with `admin` or `owner` role. Receiving clients MUST ignore this message if it is received from member with `member` role.
```

Comment:
Since there's a new "author" role
```suggestion
`x.grp.mem.role` message is sent to update group member role - it is sent to all members by the member who updated the role of the member referenced in this message. This message MUST only be sent by members with `admin` or `owner` role. Receiving clients MUST ignore this message if it is received from member with role less than `admin`.
```